hack/go-build-bundle.sh

```diff
@@ -78,7 +81,7 @@ if [[ ${LOCAL} != true ]]; then
 else
     cekit -v --descriptor image-bundle.yaml --redhat build \
         --overrides '{name: '${BUNDLE_NAME}'}' \
-        --overrides '{version: '${VERSION}'}' \
+        --overrides '{version: '${CSVVERSION}'}' \
```

Comment:
why changing this? image tag needs to be `7.11.0`, not 7.11.0-2... right?

Reply:
I have updated because of this the instructions on the README points the bundle image tag to 7.xx.y-z

Btw, before merging this I should increase the tag version, right?

Reply:
I see, will revert it back, thanks.